src/App.js

```diff
@@ -2,70 +2,122 @@ import {nanoid} from 'nanoid';
 import {useState} from 'react';
 import styled from 'styled-components';
 
+import Footer from "./components/Footer";
+import Header from "./components/Header";
 import InputDataDialog from './components/InputDataDialog.js';
 import { initialInputData } from "./db";
+import { messages } from "./db";
 
 export default function App() {
   
   const [inputs, setInputs] = useState(initialInputData);
+  //const [inputValues, setInputValues] = useState(inputs[0].value);
+  const [showmessages, setShowMessages] = useState(messages[0].text);
   
+  const oldInputLength = initialInputData.length;
+  //alert(initialInputData.length)
   const actualDate = new Date();
-  const formattedActualDate = actualDate.toLocaleDateString()
+  const formattedActualDate = actualDate.toLocaleDateString('en-GB', {year: 'numeric', month: '2-digit', day: '2-digit'})
 
   function updateInput(inputDataValue) {
+    console.log(inputDataValue)
     const newInput = {
       id: nanoid(),
       date: formattedActualDate,
-      value: inputDataValue,
+      value: Number(inputDataValue),
   };
+ 
+  if (newInput.value >= inputs[0].value)
+    {setInputs([newInput, ...inputs])
+    if (oldInputLength !== inputs.length)
+      {setShowMessages(messages[1].text)}
+    else
+      {setShowMessages(messages[2].text)}
+    }
+  else 
+    {setShowMessages("please input > or = " + inputs[0].value)}
```

Comment:
What is the purpose of this code? Your formatting is a bit off, do you use `prettier`?